eth-types/src/evm_types/opcode_ids.rs

```diff
@@ -675,6 +675,40 @@ impl OpcodeId {
                 | OpcodeId::EXTCODECOPY
         )
     }
+
+    /// Returns PUSHn opcode from parameter n.
+    pub fn push_n(n: u8) -> Result<Self, Error> {
+        if (1..=32).contains(&n) {
+            Ok(OpcodeId::try_from(OpcodeId::PUSH1.as_u8() + ((n - 1) as u8)).unwrap())
```

Comment:
`TryFrom<u8> for OpcodeId` already returns `Result<OpcodeId, Error>`. Also, `n` is already a `u8` type?

So you can replace this with:
```suggestion
            OpcodeId::try_from(OpcodeId::PUSH1.as_u8() + n - 1)
```